README.md

```diff
@@ -7,16 +7,19 @@ At 10up, we strive to provide digital products that yield a top-notch user exper
 
 ## Dependencies
 
-1. [Node & NPM](https://www.npmjs.com/get-npm) - Build packages and 3rd party dependencies are managed through NPM, so you will need that installed globally
-2. [Gulp](https://gulpjs.com/) - Gulp is used as the main task runner, it runs PostCSS, processes images, handles SVG sprites (if needed), and executes Webpack
-3. [Webpack](https://webpack.js.org/) - Webpack is used to process the JavaScript
+1. [Node & NPM](https://www.npmjs.com/get-npm) - Build packages and 3rd party dependencies are managed through NPM, so you will need that installed globally.
+2. [Gulp](https://gulpjs.com/) - Gulp is used as the main task runner, it runs PostCSS, processes images, handles SVG sprites (if needed), and executes Webpack.
+3. [Webpack](https://webpack.js.org/) - Webpack is used to process the JavaScript.
 4. [Composer](https://getcomposer.org/) - Composer is used to manage PHP.
+5. [Browsersync](https://browsersync.io/) - Browsersync is used for effortless previews of your site across your local devices.
```

Comment:
@timwright12 I have it in the other list as well, but I thought it was worth making people aware of it, as it's a useful part of the toolkit.

Reply:
@JodiWarren this is more so the list for global stuff you need (like node) installed rather than secondary tools

Reply:
Not to be overly pedantic, but should we remove Gulp and Webpack too, or maybe move them into a different section? They're not external dependencies either, and it might suggest that people need them installed globally to use the build system.

Reply:
We can deal with that in a future PR, for the purposes of this one, let's move Browsersync down to the other section.